service/models.py

```diff
@@ -126,6 +126,41 @@ def deserialize(self, data):
             ) from error
         return self
 
+    def deserialize_update(self, data):
```

Comment:
there is already have a  deserialize(self, data) func, you can reuse it.

Reply:
it's only possible to reuse if the payload contains all the columns of the product, other wise it will raise Key Error.

Reply:
i see. i find there is an warning when i use make lint. You can add this before function to ignore the flake8 warning.

```
# flake8: noqa: C901
def deserialize_update(self, data):
...
```

Reply:
thanks, I was just trying to figure out how to get rid of this warning